src/openforms/registrations/contrib/objects_api/tests/test_utils.py

```diff
@@ -0,0 +1,36 @@
+from django.test import TestCase
+
+from ..utils import escape_html_manually
+
+
+class EscapeHTMLTests(TestCase):
```

Comment:
I'd also like to see some tests with escapable keys that assert the keys are not actually escaped!